index.html

```diff
@@ -41,5 +41,193 @@ <h2>
                 </button>
             </div>
         </section>
+        <section>
+            <h1 class="work-title">My Recent Works</h1>
+            <div class="indicator">
```

Comment:
- [ ] Good work on your project cards but I would highly recommend that you position them in the center of your mobile view

![image](https://user-images.githubusercontent.com/98453979/212463622-2b967151-f23d-4815-9e4e-b999061b28ac.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/212463681-8b8269e7-3bcc-47ee-8d63-853204b34a7c.png)